### DISCUSSION TITLE
Yubikey error

### DISCUSSION THREAD

**Discussion by ralphvanos** [dani-garcia/vaultwarden#1793] [Q&A]
When trying to enable Yubikey I get the following error after I have entered my Main password.:

YUBICO_CLIENT_ID or YUBICO_SECRET_KEY environment variable is not set. Yubikey OTP Disabled.

Any suggestions?

**Comment by bokkabonga** [comment]
It would be pretty helpful to get more Information about your config. You can genereate a support string on the admin panel under "diagnostics". I also suggest you, to check the logfile for any related entries.

**Comment by ralphvanos** [comment]
You can genereate a support string on the admin panel under "diagnostics"

Where can I find this? I do not have an admin panel.

**Comment by BlackDex** [comment]
@ralphvanos you need to go to `my.domain.tld/admin` which is your admin interface

**Comment by BlackDex** [comment]
@ralphvanos do you still have this issue, or can we close this?
Also, try the testing version and see if that solves your issues.

**Comment by BlackDex** [comment]
Also, see https://github.com/dani-garcia/vaultwarden/wiki/Enabling-Yubikey-OTP-authentication
